src/app/lender/layout.tsx

```
@@ -1,12 +1,21 @@
 'use client';
 
+
+import { DashboardFooter } from "@/components/layouts/Footer";
+import { DashboardHeader } from "@/components/layouts/Header";
+import { RouteGuard } from "@/components/auth/RouteGuard";
+import { LenderSidebar } from "@/components/layouts/lender/Sidebar";
+import ParticlesBackground from "@/components/lender/active-loans/ParticlesBackground"; 
+import type React from "react";
+import { useState, useEffect } from "react";
 import { DashboardFooter } from '@/components/layouts/Footer';
```

Biome (1.9.4)

[error] Shouldn't redeclare 'DashboardFooter'. Consider to delete it or rename it.

'DashboardFooter' is defined here:


(lint/suspicious/noRedeclare)